setup-container/setup/kubelet.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Kubernetes Kubelet
+Documentation=https://github.com/kubernetes/kubernetes
+
+[Service]
+ExecStartPre=/usr/bin/mkdir -p /etc/kubernetes/manifests
+ExecStart=/usr/bin/hyperkube kubelet \
+    --allow-privileged=true \
+    --hostname-override=127.0.0.1 \
+    --address=0.0.0.0 \
+    --api-servers=http://localhost:8080 \
```

Comment:
On master/minion host you have /etc/sysconfig/heat-params file with environment variables like:
```
      MASTER_IP
      ROLE
      DOCKER_REGISTRY_URL
      DOCKER_REGISTRY_PREFIX
```